contrib/jenkins_tests/cov.sh

```diff
@@ -46,6 +46,16 @@ eval "cov-analyze --config ${cov_dir}/coverity_config.xml \
 	--dir ${cov_build}"
 rc=$(($rc+$?))
 
+if [[ "${do_coverity_snapshot}" == true ]]; then
+    cov-commit-defects --ssl --on-new-cert trust \
+        --url https://coverity.mellanox.com:8443 \
+        --user "${XLIO_COV_USER}" --password "${XLIO_COV_PASSWORD}" \
+        --dir "${cov_build}" \
+        --stream libxlio-main \
+        --strip-path "${WORKSPACE}"
+fi
+rc=$(($rc+$?))
```

Comment:
It can be meaningless here (when we dont run cov commit for example), should be in the if/fi block

Reply:
i dont see the fix ?